docs/framework/react/guides/important-defaults.md

```diff
@@ -36,7 +36,7 @@ Out of the box, TanStack Query is configured with **aggressive but sane** defaul
 
 Have a look at the following articles from our Community Resources for further explanations of the defaults:
 
-- [Practical React Query](../tkdodos-blog#1-practical-react-query)
-- [React Query as a State Manager](../tkdodos-blog#10-react-query-as-a-state-manager)
+- [Practical React Query](../../community/tkdodos-blog#1-practical-react-query)
+- [React Query as a State Manager](../../community/tkdodos-blog#10-react-query-as-a-state-manager)
```

Comment:
looking at the file structure, we are in:

`docs/framework/react/guides/important-defaults.md`

and we want to link towards

`docs/framework/react/community/tkdodos-blog.md`

so imo the correct link should be one level up:

`../community/tkdodos-blog`

this is also what my editor would auto-complete. So why are two levels up (`../../` ) needed ?

Reply:
I'm not sure why the two `..`-s are needed. I just assumed that it's related to the conversation in the [in the `#router-maintainers` channel in Discord](https://discord.com/channels/719702312431386674/1015406599738761307/1227323140925816986) about trailing slashes, and [VSCode improperly flagging links with errors](https://discord.com/channels/719702312431386674/1015406599738761307/1227343543748530217).  Also, it did not work the VSCode-way. 🤣 (Still doesn't, I checked.)

I'll ask around in Discord, and come back with the answer.

Reply:
Okay, I presented this theory to Tanner, and he said [it's correct](https://discord.com/channels/719702312431386674/1015406599738761307/1229555107473920131): 

1. the file is `docs/framework/react/guides/important-defaults.md`
2. we want to link to `docs/framework/react/community/tkdodos-blog.md`
3. the relative link that works is `../../community/tkdods.md`

My theory on why we need two `..`-s here:
- somewhere along the way the original markdown gets interpreted as the `docs/framework/react/guides/important-defaults/` route
- `.` parent is `important-defaults`
- `..` grandparent is `guides`
- `../..` great-grandparent is `react`

Hence `./../../community` takes us to the `community` route in the current path's great-grandparent route.